test/setup/make_source.sh

```diff
@@ -19,6 +19,9 @@ echo "#!/usr/bin/env bash" > ../source.sh
 project_id=$(terraform output project_id)
 echo "export TF_VAR_project_id='$project_id'" >> ../source.sh
 
+svc_project_id=$(terraform output service_project_id)
+echo "export TF_VAR_service_project_id='$svc_project_id'" >> ../source.sh
```

Comment:
@morgante I believe we can remove this now? AFAIK we use [this](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/export_tf_outputs.py) now instead automatically surfaces these.

Reply:
Thanks for pointing this out! Removed the file and it works perfectly :)